docs/content/0.1.0/docs/dj-concepts/node-dependencies.rst

```diff
@@ -1,3 +1,53 @@
 ---
 weight: 40
----
\ No newline at end of file
+---
+
+-----------------
+Node Dependencies
+-----------------
+
+Relationships between nodes are tracked by a DJ server. A node's position in the DJ DAG is determined by the node's definition,
+particularly the query. Node queries reference other DJ nodes as tables and this is what defines upstream and downstream
+dependencies for any given node. In other words--given a node, the other nodes it queries are its **upstream** dependencies
```

Comment:
you use `upstream/downstream` here but refer to parents in the bottom section

Reply:
Which do you feel is more clear? I notice we've all started using upstream/downstream in our DJ syncs but in the source code we call these missing upstream dependencies `MissingParent`s. Sounds like I should rename that class to `MissingUpstreamNode` and we use upstream/downstream everywhere in the source code and docs.

Reply:
I personally have no preference

Reply:
Updated to change "parent" to "upstream"